Assets/SEE/XR/Rotator.cs

```diff
@@ -176,12 +173,24 @@ private void GetRotationDistance(float currentAngle)
         }
 
         /// <summary>
-        /// Calculates the current rotation angle of the controller.
+        /// Checks if the rotation was significant enough (bigger than the angleTolerance).
+        /// If so, the selected object will be rotated, otherwise nothing will happen.
         /// </summary>
         /// <param name="currentAngle">The current angle of the controller.</param>
         /// <param name="startAngle">The base angle.</param>
+        /// <param name="angleTolerance">The minimum angle, which needs to be reached to cause a rotation of the selected object.</param>
         /// <returns>the amount of rotation</returns>
-        private float CheckAngle(float currentAngle, float startAngle) => (360f - currentAngle) + startAngle;
+        private bool CheckAngle(float currentAngle, float startAngle, float angleTolerance)
+        {
+            if (((360f - currentAngle) + startAngle) < angleTolerance)
+            {
+                return false;
+            }
+            else
+            {
+                return true;
+            }
```

Comment:
We can directly return the negated boolean value here:
```suggestion
            return ((360f - currentAngle) + startAngle) >= angleTolerance;
```